src/java/us/kbase/mobu/tester/ModuleTester.java

```diff
@@ -229,14 +230,16 @@ public void logNextLine(String line, boolean isError) {
                         }
                     }).build();
             ModuleRunVersion runver = new ModuleRunVersion(
-                    new URL("https://fakefakefakefakefake.com"),
-                    new ModuleMethod("use_set_provenance.to_set_provenance_for_tests"),
-                    "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa", "0.0.0", "dev");
+                    new URL("https://localhost"),
+                    new ModuleMethod(moduleName + ".to_set_provenance_for_tests"),
```

Comment:
I would remove to_set_prov... here and change to .run_local_tests or something like that.